README.md

```diff
@@ -1,25 +1,31 @@
 # preCICE adapter for MBDyn #
 
 ## Installation ##
+No installation configured, instead include folder in PYTHONPATH
 ```
-sudo python setup.py install
+export PYTHONPATH=$PYTHONPATH:<path-to-adapter>
```

Comment:
I installed MBDyn and tried to run the case by going in `example/cavity` and running `./Allrun`. I got the following error on the Solid participant:
```
Traceback (most recent call last):
  File "predyn.py", line 1, in <module>
    from adapter import MBDynAdapter
ModuleNotFoundError: No module named 'adapter'
```
is this something related to paths or installation?